test/dub_package/README.md

```diff
@@ -0,0 +1,17 @@
+DMD as a library
+================
```

Comment:
Personally I prefer the hash `#` for headers. Then I only need one character and don't have to add/remove characters if the title changes.

Reply:
Yeah I usually use a mix of these styles in my markdown: `===` for the title and `#` for all headers within the file as the title usually changes seldom, but has more emphasis this way.
Anyhow I wouldn't mind at all to use a `#` if that's important, but I think there are more pressing issues than which flavor of Markdown should be used?

Reply:
No, it's not that important. That's why I said it's my personal preference.